### PR TITLE
OCM-3277 | feat: Added open api model for trusted ips

### PR DESCRIPTION
**Pull request by den-rgb**
### [JIRA](https://issues.redhat.com/browse/OCM-3277)

### CODE REVIEW

File: model/clusters_mgmt/v1/root_resource.model [hunk 1]
```diff
@@ -111,4 +111,9 @@ resource Root {
 	locator NetworkVerifications {
 		target NetworkVerifications
 	}
+
+	// Referenceto the resource that manages the collection of trusted ip addresses
```
Comment (gdbranco) [inline model/clusters_mgmt/v1/root_resource.model:115]: typo and missing '.' at the end

File: model/clusters_mgmt/v1/trusted_ip_resource.model [hunk 2]
```diff
@@ -0,0 +1,23 @@
+/*
+Copyright (c) 2020 Red Hat, Inc.
```
Comment (gdbranco) [inline model/clusters_mgmt/v1/trusted_ip_resource.model:2]: 23

File: model/clusters_mgmt/v1/trusted_ips_resource.model [hunk 3]
```diff
@@ -0,0 +1,39 @@
+/*
+Copyright (c) 2019 Red Hat, Inc.
```
Comment (gdbranco) [inline model/clusters_mgmt/v1/trusted_ips_resource.model:2]: 23

File: model/clusters_mgmt/v1/trusted_ips_type.model [hunk 4]
```diff
@@ -0,0 +1,23 @@
+/*
+Copyright (c) 2020 Red Hat, Inc.
```
Comment (gdbranco) [inline model/clusters_mgmt/v1/trusted_ips_type.model:2]: 23

File: model/clusters_mgmt/v1/trusted_ips_type.model [hunk 5]
```diff
@@ -0,0 +1,21 @@
...
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+// Representation of a trusted ip address in clusterdeployment.
+class TrustedIp {
+	// the boolean set to show if the ip is enabled
```
Comment (zgalor) [inline model/clusters_mgmt/v1/trusted_ips_type.model:19]: '.' missing at the end of the comment

File: model/clusters_mgmt/v1/trusted_ips_type.model [hunk 6]
```diff
@@ -0,0 +1,21 @@
...
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+// Representation of a trusted ip address in clusterdeployment.
+class TrustedIp {
+	// the boolean set to show if the ip is enabled.
```
Comment (zgalor) [inline model/clusters_mgmt/v1/trusted_ips_type.model:19]: np: capital `T`

Comment (den-rgb) [reply model/clusters_mgmt/v1/trusted_ips_type.model:19]: as in `class trustedIp` ?

Comment (den-rgb) [reply model/clusters_mgmt/v1/trusted_ips_type.model:19]: or in the comment?

Comment (zgalor) [reply model/clusters_mgmt/v1/trusted_ips_type.model:19]: No no. At the beginning of the comment `The boolean...`

**Review by zgalor — APPROVED** [review]
LGTM

**Pull request merged**